### ISSUE TITLE
Suggestion on Jobs

### ISSUE THREAD

**Issue by andslasher123** [ldlac/La_LifeRP#171]
I have a suggestion the legal jobs like trash and pool can you make it so if your far away from the vehicle they fail the job because I see people using sports cars to do the job and leave the service vehicle behind....

J'ai une suggestion pour les emplois légaux comme les ordures et les bassins. Si vous êtes loin du véhicule, ils échouent parce que je vois des gens qui utilisent des voitures de sport pour faire le travail et laisser le véhicule de service derrière ...

**Comment by ldlac** [comment]
Impossible, les points ne se génèrent pas si le joueur n'est pas dans le véhicule de job en principe

**Comment by andslasher123** [comment]
that's false am doing the pool job and am doing all these missions without the job vehicle I get show you proof


C'est faux, je fais le travail de la piscine et je fais toutes ces missions sans le véhicule d'emploi, je vous montre une preuve

**Comment by andslasher123** [comment]
heres the video to show you https://youtu.be/A2fAzCVOLyA

**Comment by TonTonSheitan** [comment]
Oui, avant quand tu t'éloigner du véhicule la mission était annulé plus maintenant depuis les dernière mise a jour
